### PR TITLE
More chroma-externals references removal

### PR DESCRIPTION
**Pull request by brianjmurrell**
Revert "HYD-3860 Add git hooks to warn if the chroma externals ref. has been modified."
This reverts commit 36d5e6304f2efbc7842d697137476b53d92416da.

Remove any pre-existing chroma-externals sub-directory.

### CODE REVIEW

**Comment by jgrund** [comment]
Should we also get rid of the test_dependencies files?

**Comment by brianjmurrell** [comment]
> Should we also get rid of the test_dependencies files?

I thought I did.  Did that not make it into the commit?  I will if it didn't.

File: chroma-manager/tests/framework/utils/install_deps.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Copy the repo to the test node (don't want to have to clone every test run - impacts Gerrit)
-# We exclude chroma-externals and copy just what we need to reduce the io load on our builders.	
+# We exclude chroma-externals and copy just what we need to reduce the io load on our builders.
```
Comment (jgrund) [inline chroma-manager/tests/framework/utils/install_deps.sh]: Is this still relevant?

Comment (brianjmurrell) [inline chroma-manager/tests/framework/utils/install_deps.sh]: I will remove the reference to chroma-externals from the comment.

**Pull request closed without merging**